### PR TITLE
Specific Usage Example for README.org

### PR DESCRIPTION
**Pull request by idrozd**
Example for complete emacs/elisp newbies 

from someone who finally made theme customisation work after like ~ 10th attempt

### CODE REVIEW

File: layers/+themes/theming/README.org [hunk 1]
```diff
@@ -28,6 +28,30 @@ of the following form:
            (face2 attributes...)
            ...)
    ...)
+   
+   
+   https://gist.github.com/TheBB/f25a607b9bda4d5861f2#file-init-el-L274
```
Comment (bmag) [inline layers/+themes/theming/README.org]: Doesn't belong here, probably got here by mistake. The new empty lines are unnecessary as well.

File: layers/+themes/theming/README.org [hunk 2]
```diff
@@ -28,6 +28,30 @@ of the following form:
...
+#+end_src
+
+* Example based on TheBB's (much more broad) [[https://gist.github.com/TheBB/f25a607b9bda4d5861f2#file-init-el-L274][gist]]
```
Comment (bmag) [inline layers/+themes/theming/README.org]: 1. Should be a sub-heading of "Usage"
2. Not a good heading name. "Example" is enough, a reference to TheBB's source can be added as a comment in the example code.

File: layers/+themes/theming/README.org [hunk 3]
```diff
@@ -28,6 +28,30 @@ of the following form:
...
+#+begin_src lisp
+  ;; ~/.spacemacs
+  ;;...
```
Comment (bmag) [inline layers/+themes/theming/README.org]: `;; ...` not `;;...` (spacing is nice)

File: layers/+themes/theming/README.org [hunk 4]
```diff
@@ -28,6 +28,30 @@ of the following form:
...
+  (defun dotspacemacs/user-init ()
+  ;;...
+   (setq-default 
+     ;;... other vars maybe
```
Comment (bmag) [inline layers/+themes/theming/README.org]: unnecessary

File: layers/+themes/theming/README.org [hunk 5]
```diff
@@ -28,6 +28,30 @@ of the following form:
...
+   https://gist.github.com/TheBB/f25a607b9bda4d5861f2#file-init-el-L274
+#+end_src
+
+* Example based on TheBB's (much more broad) [[https://gist.github.com/TheBB/f25a607b9bda4d5861f2#file-init-el-L274][gist]]
+#+begin_src lisp
+  ;; ~/.spacemacs
+  ;;...
+  (defun dotspacemacs/user-init ()
+  ;;...
+   (setq-default
```
Comment (bmag) [inline layers/+themes/theming/README.org]: 1. `setq` is more suitable here
2. should have one more indent space (total indentation should be 2 spaces deeper than above), adjust lines below as necessary.

File: layers/+themes/theming/README.org [hunk 6]
```diff
@@ -28,6 +28,30 @@ of the following form:
...
+   
+   https://gist.github.com/TheBB/f25a607b9bda4d5861f2#file-init-el-L274
+#+end_src
+
+* Example based on TheBB's (much more broad) [[https://gist.github.com/TheBB/f25a607b9bda4d5861f2#file-init-el-L274][gist]]
+#+begin_src lisp
+  ;; ~/.spacemacs
+  ;;...
+  (defun dotspacemacs/user-init ()
+  ;;...
```
Comment (bmag) [inline layers/+themes/theming/README.org]: `;; ...` is nicer and should have the same indent as `(setq` below

File: layers/+themes/theming/README.org [hunk 7]
```diff
@@ -28,6 +28,30 @@ of the following form:
...
+      '((monokai
+          ;; Font locking
+          (font-lock-comment-face :slant italic)
+          (web-mode-html-attr-name-face
+           :inherit font-lock-variable-name-face :foreground nil)
+      
+          ;; Modeline
+          (powerline-active1 :box (:color "#999999" :line-width 1 :style released-button)
+                             :background "#5a5a5a"))))
+  )
```
Comment (bmag) [inline layers/+themes/theming/README.org]: Would be nicer to move this paren to the previous line and join it to the other parens.

**Comment by bmag** [comment]
Note that the build is failing because you submitted the PR against master branch instead of develop branch

**Comment by syl20bnr** [comment]
Thank you !
I applied @bmag's suggestions.
Cherry-picked into develop branch, you can safely delete your branch.

**Pull request closed without merging**